module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -274,10 +292,19 @@ protected function storeSanitizedSettings($settings)
         } else {
             $this->issn = null;
         }
+
+        if (!empty($settings['ismn'])) {
+            $rawismn = preg_replace('/[^0-9X]/', '', strtoupper($settings['ismn']));
```

Comment:
It looks like ISMN codes come in 10- and 13-digit forms and are in some ways compatible with ISBNs; can the VuFindCode ISBN logic be applied here and/or does it need to be extended to include ISMN support?

Reply:
In 13 digit form, the ISMN has the same format as ISBN, only diference is in prefix, as ISBN is prefixed by '978' and ISMN by '979-0'. In 10 digit form (used before 2008) the ISMN looks  like M69200-628-2 , so the number is one digit shorter and is prefixed by 'M'. So this should be reflected in VuFindCode.

Reply:
For now, I've just fixed the regexp...

Reply:
Thanks. I have opened an issue on VuFindCode to track the possible desire to add ISMN support: https://github.com/vufind-org/vufindcode/issues/5 -- I'm not sure if/when I will have time for this, but I've linked it to this PR as a reminder to make use of it here if we implement it there.